styles.css

```diff
@@ -565,3 +676,551 @@ a {
   background-color: #fff;
   color: #7f8cff;
 }
+
+@media screen and (min-width: 1200px) {
```

Comment:
- [ ] Please, use media query for the Desktop breakpoint in the CSS file.(min-width > 992px - 1100px)
```html
// Doesn't pass the requirement (too small)
@media screen and (min-width: 990px) {
/*
.
.

*/
}

// Passes the requirement (propper size)
@media screen and (max-width: 1080px) {
/*
.
.

*/
}
```

Reply:
Rectified the media query to @media screen and (min-width: 992px) because it will take into consideration widths between 992-1100 and more.